components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/Oauth2ScopeUtils.java

```diff
@@ -271,4 +273,26 @@ private static boolean iterateOAuth2ScopeValidators(OAuthAuthzReqMessageContext
         }
         return true;
     }
+
+    /**
+     * Read RoleBasedScopeValidation configurations for Implicit Grant from identity.xml file.
+     *
+     * @param xPath xpath of the config property.
+     * @return Config property value.
+     */
+    public static boolean readRoleBasedScopeValidationEnabledConfigValue(String xPath) {
```

Comment:
We usually get OAuth related configurations from OAuthServerConfiguration object. Shouldn't we do the same here as well?

Reply:
done